e2e/testcafe-devextreme/tests/dataGrid/common/tagBox.ts

```diff
@@ -67,3 +69,31 @@ fixture.disablePageReloads`Tagbox Columns`.page(
       });
   },
 );
+
+test('TagBox - Methods specified in DevExpress.ui.dxSelectBox.defaultOptions are called (T1286533)', async (t) => {
```

Comment:
Let's move this file, it tests only tagBox component, it shouldn't be located inside dataGrid folder
Ideally, there should be test for abstract component somewhere inside qunit tests. We already have them, please check